<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>20</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Новичок 8х8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Любитель 16х16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Профессионал 30х16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Настраиваемый </string>
    </property>
   </item>
  </widget>
  <widget class="QTableWidget" name="tw_top">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>741</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_btn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
